lib/peep.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+class Peep
```

Comment:
Refactoring the Peep class would be desirable as it has too many responsibilities. Perhaps a DatabaseConnection class.